### ISSUE TITLE
Make SVGs Accessible

### ISSUE THREAD

**Issue by adhagam** [pricelinelabs/design-system#477]
- Default title tag included in all SVG icons
- <title> should be configurable
- aria-labelledBy and aria-hidden tags should be included and configurable

**Comment by adhagam** [comment]
Added Accessibility support for SVG icons. Issue Resolved
